cc/main.cc

```diff
@@ -145,6 +151,17 @@ std::string GetOutputName(absl::Time now, size_t i) {
   return absl::StrCat(timestamp, "-", hostname, "-", i);
 }
 
+std::string GetRowPrefix(absl::Time now, int id) {
+  std::ostringstream out;
+  absl::TimeZone utc = absl::UTCTimeZone();
+  out << absl::FormatTime("%Y%m%d-%H%M%S", now, utc)
+      << "-w"
+      << std::setw(3)
```

Comment:
Use 4 digit padding (or even higher): Christian Sigg has been experimenting with up to 2048 parallel games.

Reply:
Cool! Will use 6-digit padding, because a million is a nice round number.